### PR TITLE
BAH 1033 - Adding pom.xml changes to add Appointments 1.2 to sonatype

### PR DESCRIPTION
**Pull request by buvaneswari-arun**
pom.xml changes are done.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,17 +74,37 @@
...
-					<target>1.8</target>
-				</configuration>
-			</plugin>
-		</plugins>
+<!--		<plugins>-->
+<!--			<plugin>-->
+<!--				<artifactId>maven-compiler-plugin</artifactId>-->
+<!--				<configuration>-->
+<!--					<source>1.8</source>-->
+<!--					<target>1.8</target>-->
```
Comment (praveenadayanand) [inline pom.xml:104]: Why is this code commented and not removed ?

File: pom.xml [hunk 2]
```diff
@@ -243,11 +287,10 @@
...
 
 	<distributionManagement>
-		<snapshotRepository>
-			<id>repo.mybahmni.org</id>
-			<name>bahmni-artifactory-snapshots</name>
-			<url>s3://repo.mybahmni.org/artifactory/snapshot</url>
-		</snapshotRepository>
+			<snapshotRepository>
+						<id>nexus-sonatype</id>
+						<url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (praveenadayanand) [inline pom.xml:292]: Suggestion: apply proper formatting

**Comment by binduak** [comment]
@buvaneswari-arun can you sign the CLA to proceed with the merge ??

File: pom.xml (hunk 1 above)
Comment (buvaneswari-arun) [reply pom.xml:104]: This plugin has been repeated twice, once inside plugin management and once outside, not sure of the usecase for which this has been implemented this way. Hence commented it out.

File: pom.xml (hunk 2 above)
Comment (buvaneswari-arun) [reply pom.xml:292]: Will do the same

**Pull request merged**